### ISSUE TITLE
General keyboard shortcuts for modal

### ISSUE THREAD

**Issue by tallulahkay** [civiform/civiform#12757]
In Chrome, with VoiceOver, using keyboard navigation, focus is on elements in the background.  Focus should be trapped in the modal until it is dismissed.

[Slack Message](https://civiform.slack.com/archives/C07KXU02B70/p1771435475624969?thread_ts=1771435475.624969&cid=C07KXU02B70)
